src/components/Questionnaire/QuestionTypes/ConsentQuestion.tsx

```
@@ -0,0 +1,436 @@
+    const fieldErrors = validateFields(
+      {
+        [CONSENT_FIELDS.FILES.key]: value.files,
+      },
+      questionId,
+      CONSENT_FIELDS,
+      index,
+    );
+
+    // Map error messages to be more specific
+    return [
+      ...errors,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)